priv/worker.py

```diff
@@ -33,7 +49,7 @@
     year = {2016},
     publisher = {GitHub},
     journal = {GitHub repository},
-    howpublished = {\url{https://github.com/openai/gym-http-api}}
+    howpublished = {url{https://github.com/openai/gym-http-api}}
```

Comment:
yes because the decoder thinks we are using a unicode here \UXXXXXXXX which is invalid character